src/clj/harja/palvelin/integraatiot/velho/varusteet.clj

```diff
@@ -178,8 +211,8 @@
                       " version-alku:" version-alku ". Toimenpiteet: " (str/join ", " toimenpiteet)
                       " Otetaan vain 1. toimenpide talteen.")))
         (:otsikko (first (memoized-hae-nimikkeen-tiedot db {:tyyppi-nimi (first toimenpiteet)}))))
-
       (cond
+        (seq valimaiset-toimenpiteet) (str/join "," (keep #(:otsikko (first (memoized-hae-nimikkeen-tiedot db {:tyyppi-nimi %}))) valimaiset-toimenpiteet))
```

Comment:
Luettavuus hieman kaaoottista, ehdotuksena voisi heittää pari line breakkia ja vaikka anonyymi funktion väliin, eli jotenkin näin? 

```clojure
(cond
  ;; Välimmäiset toimenpiteet?
  (seq valimaiset-toimenpiteet)
  (str/join ","
    (keep
      (fn [toimenpide]
        (:otsikko
         (first (memoized-hae-nimikkeen-tiedot db {:tyyppi-nimi toimenpide})))) 
      valimaiset-toimenpiteet))
  ;; Poistettu? 
  (some? poistettu?) "Poistettu"
  (or
    (and (nil? version-voimassaolo) alkaen (not poistettu?))
    (and alkaen version-alku (not version-loppu) (not poistettu?))) "Lisätty"
  ;; Else
  :else "Päivitetty")
```

Reply:
Totta aika pitkä one-liner tuossa jo joten eriytin sen omaksi funktioksi ihan.